src/extension.ts

```diff
@@ -76,6 +69,70 @@ export function activate(context: vscode.ExtensionContext) {
             }
         }
     }, null, context.subscriptions);
+    
+    vscode.workspace.onDidChangeConfiguration(event => {    
+        if (event.affectsConfiguration("numberedBookmarks.gutterIconFillColor") 
+            || event.affectsConfiguration("numberedBookmarks.gutterIconNumberColor")
+        ) {
+            context.globalState.update(
+                STATE_SVG_VERSION, 
+                getCurrentSvgVersion() + 1
+            );
+            updateBookmarkSvg();  
+            updateBookmarkDecorationType();      
+        }
+    });
+    
+    // The only way to update the decorations after changing the color is to create a new file
+    function updateBookmarkSvg() {  
+        const v = getCurrentSvgVersion();
+        
+        if (fs.existsSync(context.asAbsolutePath(`images/bookmark0-${v}.svg`))) {
+            return;
+        }
+        
+        const gutterIconFillColor = getFillColor();
+        const gutterIconNumberColor = getNumberColor();
+        const content = fs.readFileSync(context.asAbsolutePath("images/bookmark.svg"), "utf8");
+        
+        for (let i = 0; i <= 9; i++) {
+            const svgContent = content
+                .replace("{{gutterIconFillColor}}", gutterIconFillColor)
+                .replace("{{gutterIconNumberColor}}", gutterIconNumberColor)
+                .replace("{{number}}", i.toString());
+                
+            try {    
+                fs.writeFileSync(context.asAbsolutePath(`images/bookmark${i}-${v}.svg`), svgContent, "utf8"); 
+            } catch (err) {
+                vscode.window.showErrorMessage(`Can't write to ${err.path}`);            
+            }
+                
+            fs.unlink(context.asAbsolutePath(`images/bookmark${i}-${v - 1}.svg`));
```

Comment:
This code is raising a **Warning** in debug console: `(node:14844) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.`. It needs to be replaced by `unlinkSync`

Reply:
fixed